### ISSUE TITLE
FullyQualifiedGlobalConstants false positive

### ISSUE THREAD

**Issue by mathroc** [slevomat/coding-standard#1367]
This sniff is confused by some namespace group use syntax. e.g. with this file:

```php
<?php

namespace N;

use Foo\{Bar, Taz};
```

`phpcs -s` says:

```
 5 | ERROR   | [x] Constant Bar should be referenced via a fully qualified name.
   |         |     (SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants.NonFullyQualified)
 5 | ERROR   | [x] Constant Taz should be referenced via a fully qualified name.
   |         |     (SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants.NonFullyQualified)
```

and phpcbf add `\` before them

**Comment by kukulich** [comment]
Yes, we don’t support group syntax for uses.
